lib/gui/models/supported-formats.js

```diff
@@ -131,6 +131,24 @@ SupportedFormats.service('SupportedFormatsModel', function() {
     return this.isSupportedImage(path.basename(image, `.${extension}`));
   };
 
+  /**
+   * @summary Check if an image seems a Windows image
+   * @function
+   * @public
+   *
+   * @param {String} image - image path
+   * @returns {Boolean} whether the image seems to be a Windows image
+   *
+   * @example
+   * if (SupportedFormatsModel.seemsWindowsImage('path/to/en_windows_7_ultimate_with_sp1_x86_dvd_u_677460.iso')) {
+   *   console.log('Looks like a Windows image');
+   * }
+   */
+  this.seemsWindowsImage = (image) => {
```

Comment:
Maybe we can refactor this function to be more generic? E.g: to allow us to see if we should display a warning for *any* image that requires special treatments, and not only Windows (for example we can extend it to detect VMWare images, which are also non-bootable without extra treatment)?

I guess we can make this function return an object containing a warning message, a warning title, maybe a documentation link, etc; or `null`/`undefined` if there is nothing to show. Then, the controller making use of this function can check if a warning was returned, and pass all these values to `WarningModalService` is a generic way.

What about something like this?

```js
this.getImageWarning = (imagePath) =>
    const basename = path.basename(imagePath);

    if (/^.+_windows_.+(?:x86|x64|ia64|arm)_dvd_.+$/.test(basename)) {
        return {
            message: messages.warning.possiblyWindowsImage(),
            confirmationLabel: '....',
            ...
        };
    }
};
```